examples/atlas-organization/organization-step-2/Readme.md

```diff
@@ -0,0 +1,64 @@
+# Example - A basic example to start with the MongoDB Atlas and Terraform
+
+This project aims to provide a very straight-forward example of setting up Terraform with MongoDB Atlas. This will create the following resources in MongoDB Atlas:
```

Comment:
Should we not explain how to get the key pair from step one to step 2?  Again that calls for an overarching readme.